conf.d/done.fish

```diff
@@ -258,8 +258,10 @@ if set -q _done_enabled
 
                 # make notification auto-disappear
                 set -l transient ""
-                if test "$__done_notification_transient" -eq 1
-                    set transient --hint=int:transient:1
+                if count (__done_notification_transient) > /dev/null
```

Comment:
Perhaps better to do it the recommended way?
https://fishshell.com/docs/current/faq.html#how-do-i-check-whether-a-variable-is-defined

Reply:
Sure!